safe-global/safe-wallet-web
# feat: [Counterfactual] Show pending status on dashboard

## What it solves

Resolves #3494

## How this PR fixes it

- Shows a different title and a loading spinner on the dashboard when activating a safe
- Shows a Pending status widget on the dashboard when activating a safe
- Shows a Useful Information widget on the dashboard when activating a safe

## Out of scope

Not included in this PR but coming later is an additional screen in case the activation is taking too long and an option to speed-up the transaction. See [Figma](https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=10822%3A2654&mode=dev) design.

## How to test it

1. Create a counterfactual safe
2. Go to the dashboard
3. Activate the safe
4. Observe a new loading state

## Screenshots
<img width="1269" alt="Screenshot 2024-03-28 at 12 05 00" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/7f0f9026-8ff4-4239-8959-6eaae039251e">

## Checklist
